actions/playerHeroesActions.js

```diff
@@ -1,8 +1,10 @@
 import fetch from 'isomorphic-fetch';
 import { HOST_URL } from './';
 import { playerHeroes } from '../reducers';
+import { getUrl, addQueryString, defaultOptions, getModifiedOptions } from './utility';
 
-const url = (playerId, queryString) => `/api/players/${playerId}/heroes?${queryString}`;
+const excludedOptions = ['limit'];
```

Comment:
It should probably be user's choice.  Maybe they want the heroes played in their last 100 games or something